README.md

```diff
@@ -1,143 +1,415 @@
-- [Other Concepts](#other-concepts)
-    - [Conversion vs. Adoption](#conversion-vs-adoption)
-    - [Value capture vs. Value creation](#value-capture-vs-value-creation)
-    - [Community activation](#community-activation)
-    - [Groups](#groups)
-    - [1:few:many](#1fewmany)
-    - [Using orbit + funnel together](#using-orbit--funnel-together-)
-- [Early Adopters](#early-adopters)
-- [Contributing](#contributing) 
 
+- [Introduction](#introduction)
+  - [Love](#love)
+  - [Reach](#reach)
+  - [Gravity](#gravity)
+- [Orbit levels](#orbit-levels)
+  - [Orbit 4 - Observers](#orbit-4-observers)
+  - [Orbit 3 - User](#orbit-3-users)
+  - [Orbit 2 - Fans](#orbit-2-fans)
+  - [Orbit 1 - Ambassadors](#orbit-1-ambassadors)
+  - [Orbit 0 - Team](#orbit-0-team)
+- [Using the levels](#using-the-levels)
+  - [Choosing levels](#choosing-levels)
+  - [Level distribution](#level-distribution)
+  - [Activities by level](#activities-by-level)
+  - [Promotions](#promotions)
+- [Orbit KPIs](#orbit-kpis)
+- [Full Example](#full-example)
+- [Additional topics](#additional-topics)
+  - [Groups](#groups)
+  - [Orbit vs. funnel](#orbit-vs-funnel)
+- [Resources](#resources)
+- [Early adopters](#early-adopters)
+- [About the Orbit Model](#about-the-orbit-model)
+- [Contributing](#contributing)
+- [Changelog](#changelog)
+
+# Introduction
+
+The Orbit Model is a vocabulary and visual canvas that helps organizations understand, manage, and grow their community. Questions like:
+
+- How do I measure my community's engagement and growth?
+- How do I attract new people to my community?
+- Which members should I prioritize spending time with?
+- What contribution should I ask each community member to make?
+
+Who can use the Orbit Model?
+
+- Developer communities
+- Open source projects
+- Community-driven businesses
+- and more...
+
+### The solar system
+
+A solar system, with all of its celestial bodies and concentric orbits, lends itself well to visualizing a community. Some members are close to the heart of things, others far away. Some members are as large and visible as Jupiter, others barely the size of an asteroid. And of course, everything is constantly in motion.
+
+To make it easy to discuss the differences between members, so we can give them the appropriate experience, the Orbit Model assigns each person 2 core attributes: _love_ and _reach_.
+
+**The goal is to increase Love and Reach for each of your community members.**
+
+## Love
+
+Love is a member's level of engagement and investment in the community. Someone with high love is highly active, capable, and experienced. They're willing and able to play key roles in the community like contributing, moderating, and organizing.
+
+> "Jennifer has come to the last 5 meetups and next month she's hosting her own. In addition, she just published a blog post about how to install our package on Linux. Her love is increasing fast this year!"
```

Comment:
> Her love is increasing fast this year!

Can this example be more specific or actionable? Maybe "Let's feature her work on the Community Showcase site," or, "Let's see if she'd be willing to speak at our conference."

Reply:
Added something